client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -298,6 +298,25 @@ function getNotes(
               label: <FlagsNote browser={browser} supportItem={item} />,
             }
           : null,
+        item.partial_implementation
+          ? {
+              iconName: "none",
+              label: "Partial support",
+            }
+          : null,
+        item.version_removed
+          ? {
+              iconName: "none",
+              label: "Support removed",
+            }
+          : null,
+        // if we only have the required version_added and releaseDate properties, assume full support
+        Object.keys(item).length === 2
```

Comment:
The code comment doesn't match the implementation. A more accurate implementation, assuming the code comment is correct, would be something like this:
```js
isEqualSet(new Set(Object.keys(item)), new Set(['version_added', 'releaseDate']))
```

Or, perhaps it would be a more "sustainable" implementation to do something like this:

```js
Object.keys(item).includes('version_added') && Object.keys(items).includes('releaseDate')
```

Reply:
Right, I was too lazy to actually compare anything and I thought it would be enough to rely on the length. Fixed.